### PR TITLE
benchmarks: optimize trie layer

### PR DESCRIPTION
**Pull request by aramikm**
# Goal
The goal of this PR is to optimize benchmark weights by isolating the **stateful-storage** to have a higher value for `additional-trie-layers` due to using of child trees instead of applying it to all pallets.

Related to #2093 

# Checklist
- [x] Benchmarks added?
- [x] Spec version incremented?

### CODE REVIEW

**Review by JoeCap08055 — APPROVED** [review]

**Review by JoeCap08055 — APPROVED** [review]
Seems pretty straightforward; lgtm.

**Review by saraswatpuneet — APPROVED** [review]
Looks good but why don't I see passkey weights 

I see it's a very old PR

File: scripts/run_benchmarks.sh [hunk 1]
```diff
@@ -208,15 +208,20 @@ for external_pallet in "${EXTERNAL_PALLETS[@]}"; do
...
 done
 
 for pallet_name in "${CUSTOM_PALLETS[@]}"; do
   steps=20
   repeat=10
   template=${PROJECT}/.maintain/frame-weight-template.hbs
   output=${PROJECT}/pallets/${pallet_name/_/-}/src/weights.rs
-  run_benchmark pallet_${pallet_name} ${steps} ${repeat} ${output} ${template} || exit_err
+  additional_trie_layer=3
+  if [ "$pallet_name" == "stateful-storage" ]; then
```
Comment (enddynayn) [inline scripts/run_benchmarks.sh:221]: 👍

**Review by enddynayn — APPROVED** [review]
👍 great!

**Review by shannonwells — APPROVED** [review]
I did not read carefully through the weights but I looked at the script changes, and `make benchmarks-local` passed.

**Comment by aramikm** [comment]
Moved to https://github.com/frequency-chain/frequency/pull/2128

**Pull request closed without merging**